server/lib/src/constants/schema.rs

```diff
@@ -1055,4 +1078,16 @@ pub static ref SCHEMA_CLASS_OAUTH2_RS_PUBLIC_DL4: SchemaClass = SchemaClass {
     ..Default::default()
 };
 
+pub static ref SCHEMA_CLASS_APPLICATION: SchemaClass = SchemaClass {
+    uuid: UUID_SCHEMA_CLASS_APPLICATION,
+    name: EntryClass::Application.into(),
+
+    description: "The class representing an application".to_string(),
+    systemmust: vec![Attribute::Name.into()],
+    systemmay: vec![Attribute::Description.into(), Attribute::LinkedGroup.into()],
+    systemsupplements: vec![EntryClass::ServiceAccount.into()],
+    systemexcludes: vec![EntryClass::Person.into()],
```

Comment:
Fixed, but Person must exclude application then. Otherwise creating
```
let e1 = entry_init!(
            (Attribute::Class, EntryClass::Object.to_value()),
            (Attribute::Class, EntryClass::Account.to_value()),
            (Attribute::Class, EntryClass::Application.to_value()),
            (Attribute::Class, EntryClass::Person.to_value()),
            (Attribute::Name, Value::new_iname("test_app_name")),
            (Attribute::Uuid, Value::Uuid(test_entry_uuid)),
            (Attribute::Description, Value::new_utf8s("test_app_desc")),
            (
                Attribute::DisplayName,
                Value::new_utf8s("test_app_dispname")
            )
        );
```
succeeds.